src/app/services/main-form-manager/config-export-helper.ts

```diff
@@ -686,17 +687,19 @@ export class ConfigExportHelper {
     }
   }
 
-  private static getChipsearchContributor(searchConfigGlobal: SearchGlobalFormGroup, collection: string): ContributorConfig {
-    return {
-      type: CHIPSEARCH_TYPE,
-      identifier: CHIPSEARCH_IDENTIFIER,
-      collection,
-      search_field: searchConfigGlobal.customControls.searchField.value,
+  private static getChipsearchContributor(searchConfigGlobal: SearchGlobalFormGroup, collection: string): ContributorConfig[] {
```

Comment:
Shouldn't this method be renamed since we are no longer handling ChipSearchContributors ?